components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/config/ConsentManagementConfigurationService.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.consent.management.common.config;
+
+import java.util.Map;
+
+/**
+ * Interface to expose Configurations as an OSGi Service.
+ */
+public interface ConsentManagementConfigurationService {
+
+    Map<String, Object> getConfigurations();
```

Comment:
It seems like we might only need the `getConfigElementFromKey` method for our purposes.  Is there a reason we're currently passing the entire configuration map around?

Reply:
we are using this method to get the full config and  register as a service in OSGI level startup